chaincode/traintuple.go

```diff
@@ -210,20 +210,24 @@ func (traintuple *Traintuple) Save(db *LedgerDB, traintupleKey string) error {
 // -------------------------------------------------------------------------------------------
 
 // createTraintuple adds a Traintuple in the ledger
-func createTraintuple(db *LedgerDB, args []string) (map[string]string, error) {
+func createTraintuple(db *LedgerDB, args []string) (out outputTraintuple, err error) {
```

Comment:
Why do you go back to named return values?

Reply:
No reason, I will restore the unnamed return values.